coalib/results/result_actions/GeneratePatchesAction.py

```diff
@@ -0,0 +1,72 @@
+import os
+from pygments.lexers import guess_lexer_for_filename
+from coalib.results.result_actions.ResultAction import ResultAction
+from coalib.results.result_actions.ShowPatchAction import ShowPatchAction
+from coalib.collecting.Collectors import (
+    collect_bears, collect_bears_by_aspects)
+from coalib.output.printers.LogPrinter import LogPrinter
+from coalib.parsing.FilterHelper import FilterHelper
+from coalib.parsing.DefaultArgParser import default_arg_parser
+from pyprint.ConsolePrinter import ConsolePrinter
+from coalib.coala_modes import (
+    mode_format, mode_json, mode_non_interactive, mode_normal)
+
+POSSIBLE_BEARS = {'SpaceConsistencyBear': 'Whitespaces', 'PEP8Bear': 'PEP'}
+
+
+def show_possibilities(console_printer, i, action):
+    console_printer.print('[{:>5}]  {}. Apply patch (\'{}\')'.format('', i,
+                                                                     action))
+
+
+class GeneratePatchesAction(ResultAction):
+    SUCCESS_MESSAGE = 'Patch applied successfully.'
+
+    is_applicable = staticmethod(ShowPatchAction.is_applicable)
+
+    def apply(self,
+              result,
+              original_file_dict,
+              file_diff_dict):
+        """
+        (G)enerate patches
+        """
+        console_printer = ConsolePrinter()
+        bears_class = []
+        possible_options = {}
+        args = default_arg_parser().parse_args()
+        args.files = []
+        log_printer = LogPrinter()
+        to_filename = None
+
+        for filename, this_diff in sorted(result.diffs.items()):
+            to_filename = this_diff.rename if this_diff.rename else filename
+            to_filename = '/dev/null' if this_diff.delete else to_filename
+            original_file_dict[filename]
+        language = guess_lexer_for_filename(to_filename, 'Hello').name
+        path = to_filename
+        path = path.rstrip(os.sep)
+        os.path.basename(path)
+
+        args.no_config = True
+        filtered_bears = FilterHelper.get_filtered_bears(
+            'language', [language], None)
+        filtered_bears = filtered_bears[0]
+        filtered_bears = filtered_bears['cli']
+        for b in filtered_bears:
+            if b.name in POSSIBLE_BEARS.keys():
+                possible_options[b.name] = POSSIBLE_BEARS[b.name]
+
+        for i, action in enumerate(possible_options, 1):
+            show_possibilities(console_printer, i, action)
+
+        choose_action = int(input('[{:>5}]  Enter a number: '.format('')))
+        list_of_bears = list(possible_options.keys())
+
+        args.default_actions = '*: ApplyPatchAction'
+        bears_class.append(list_of_bears[choose_action - 1])
+        args.no_config = True
```

Comment:
It is already set to `True` in L51 https://github.com/coala/coala/pull/4634/files#diff-adce0bc484df7b1f0cf40333a8dfae54R51